### PR TITLE
Erdpy tickets

### PR DESCRIPTION
**Pull request by claudiu725**
- Add log with the wasm output path after building a contract: `WASM file generated: ./relative/path/to.wasm`
- Add explorer logs with URLs for contract deployment and normal transactions:
  - `View this contract address in the Elrond Testnet Explorer: https://testnet-explorer.elrond.com/accounts/...`
  - `View this transaction in the Elrond Testnet Explorer: https://testnet-explorer.elrond.com/transactions/...`

### CODE REVIEW

**Comment by claudiu725** [comment]
Reverted the change regarding the path of the config.

File: erdpy/utils.py [hunk 1]
```diff
@@ -230,3 +230,23 @@ def breakpoint():
...
+def log_explorer(chain, name, path, details):
+    networks = {
+        "1": ("Elrond Mainnet Explorer", "https://explorer.elrond.com"),
+        "T": ("Elrond Testnet Explorer", "https://testnet-explorer.elrond.com"),
+        "D": ("Elrond Devnet Explorer", "https://devnet-explorer.elrond.com"),
+    }
+    if chain not in networks:
+        return
+    explorer_name, explorer_url = networks[chain]
+    logger.info(f"View this {name} in the {explorer_name}: {explorer_url}/{path}/{details}")
```
Comment (camilbancioiu) [inline erdpy/utils.py:244]:
```suggestion
    try:
        explorer_name, explorer_url = networks[chain]
        logger.info(f"View this {name} in the {explorer_name}: {explorer_url}/{path}/{details}")
    except KeyError:
        return
```

File: erdpy/projects/core.py [hunk 2]
```diff
@@ -36,8 +37,10 @@ def build_project(directory: str, options: Dict[str, Any]):
 
     guards.is_directory(directory)
     project = load_project(directory)
-    project.build(options)
+    output_wasm_file = project.build(options)
     logger.info("Build ran.")
+    relative_wasm_path = os.path.join(".", output_wasm_file.relative_to(os.getcwd()))
```
Comment (camilbancioiu) [inline erdpy/projects/core.py:42]: Please rewrite using `pathlib.Path` instead of `os.path`.

**Review by camilbancioiu — CHANGES_REQUESTED** [review]

File: erdpy/projects/core.py (hunk 2 above)
Comment (claudiu725) [reply erdpy/projects/core.py]: done

**Review by camilbancioiu — DISMISSED** [review]

**Review by camilbancioiu — APPROVED** [review]

**Review by andrei-marinica — APPROVED** [review]

**Pull request merged**